libraries/aws/provisioning/src/aws_iot_provisioning_serializer.c

```diff
@@ -384,6 +384,11 @@ AwsIotProvisioningError_t _AwsIotProvisioning_SerializeCreateKeysAndCertificateR
                                                                *pSerializationBuffer,
                                                                *pBufferSize );
 
+    if( status != AWS_IOT_PROVISIONING_SUCCESS )
+    {
+        IOT_GOTO_CLEANUP();
```

Comment:
goto seems unnecessary here. Is this the same one as yesterday?

Reply:
Oops. Missed the comment. Resolved in https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/836